sdk/metricsadvisor/ai-metrics-advisor/review/ai-metrics-advisor.api.md

```diff
@@ -553,71 +524,32 @@ export type FeedbackQueryTimeMode = "MetricTimestamp" | "FeedbackCreatedTime";
 export type FeedbackType = "Anomaly" | "ChangePoint" | "Period" | "Comment";
 
 // @public
-export type GetAlertConfigResponse = AnomalyAlertConfiguration & {
-    _response: coreHttp.HttpResponse & {
-        bodyAsText: string;
-        parsedBody: any;
-    };
-};
+export type GetAlertConfigResponse = AnomalyAlertConfiguration;
```

Comment:
Just a note: if possible (without risking doc team's work), remove these response types and use result type directly.